### DISCUSSION TITLE
windows/mac exe version consept

### DISCUSSION THREAD

**Discussion by acowsays** [Tkd-Alex/Twitch-Channel-Points-Miner-v2#240] [Ideas]
DOPE or NOPE
![gimp-2 10_Kc4nmdjjsm](https://user-images.githubusercontent.com/47616873/127593850-f517728f-f3bc-44bd-a5bd-f2b1c78fe279.png)

moo

**Comment by Tkd-Alex** [comment]
:confused: :question:
